### PR TITLE
Added python scripts for brownian motion

### PR DESCRIPTION
**Pull request by rasmusdurge**

### CODE REVIEW

**Comment by l0gren** [comment]
cool

File: MathModels/brownianBridge.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#create a brownian bridge from A to B
+import numpy
+import six
+from matplotlib import pyplot
+
+
+
+
+seed = 0
```
Comment (l0gren) [inline MathModels/brownianBridge.py:9]: idk

**Review by MIDO-96 — APPROVED** [review]
works fine

**Review by MiztaOak — CHANGES_REQUESTED** [review]
Just for testing

**Review by jonnyfre — APPROVED** [review]
ok

**Pull request merged**